File: java/spring-arch-unit/spring-arch-unit-mvc-example/src/main/java/com/bmarques/springarchunitmvcexample/repository/InvoiceRepository.java
package com.bmarques.springarchunitmvcexample.repository;

import com.bmarques.springarchunitmvcexample.domain.invoice.Invoice;
import org.springframework.stereotype.Repository;

import java.time.LocalDate;

@Repository
public class InvoiceRepository {

    public Invoice getInvoice() {
        return Invoice.builder()
                .id(1)
                .installment(LocalDate.now())
                .value(5D)
                .build();
    }
}

File: kotlin/spring-arch-unit/spring-arch-unit-mvc-example/src/main/kotlin/com/bmarques/springarchunitmvcexample/repository/InvoiceRepository.kt
package com.bmarques.springarchunitmvcexample.repository

import com.bmarques.springarchunitmvcexample.domain.address.Street
import com.bmarques.springarchunitmvcexample.domain.customer.Customer
import com.bmarques.springarchunitmvcexample.domain.invoice.ElectronicInvoice
import java.time.LocalDate

class InvoiceRepository {

    fun getInvoice(): ElectronicInvoice =
            ElectronicInvoice(1, LocalDate.now(), 50.0, Customer(1, "Tiago", Street("Jose Marcos Tome", 75)))
}